### ISSUE TITLE
Accessibility Release test pass for TSDoc - 02/04/2022 (A11ytest_tracking, Feature_Task) 

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#44]
8: [Playground]: Text spacing is not getting adopted to menu items in ‘Choose a sample’ and 'Light Theme' combo box.[Sev3][WCAG1.4.12][WCAG2.1]
19: [Intro]:Keyboard focus is not visible on first interactive element on activating the 'Intro' link.[Sev3][WCAG2.4.3]
21: [TSDoc Home page]: Horizontal scroll bar appears in 'Code' part when text spacing is enabled.[Sev3][WCAG1.4.12][WCAG2.1]
30: [Accessibility] [Screen reader -Search Docs]: Screen reader is not giving the information about displayed suggestions. [Sev2][WCAG1.3.1] 
31: [Accessibility] [Screen reader -Intro]: Name is not properly defined for ‘Intro’. [Sev3][WCAG4.1.2]
32: [Accessibility] [Supporting the platform -Intro]: user is not able to access the scroll bar in code part when the page is zoomed to 200%. [Sev3][WCAG1.4.4] 
33: [Accessibility] [Screen reader -Tags]: Table is not implemented properly in ‘Tags’ page. [Sev3][MAS1.3.1]
34: [Accessibility] [Supporting the platform -GitHub]: Horizontal scroll bar appears when the page is zoomed to 400%. [Sev3][WCAG1.4.10]
35: [Accessibility] [Supporting the platform -Playground]: Page content is not visible properly when the page is zoomed to 400%. [Sev2][WCAG1.4.10]
36: Accessibility] [Keyboard Navigation -Playground]: Focus order is not logical after selecting the tabs in playground page. [Sev3][WCAG2.4.3]
37: Accessibility] [Screen reader -Playground]: Incorrect role as link is defined for the tabs. [Sev2][WCAG4.1.2] 
38: Accessibility] [Keyboard Navigation -Playground]: Keyboard focus struck in Code pane in playground page. [Sev2][WCAG2.1.2]
39: Accessibility] [Screen reader -Playground]: Main landmark is not defined in the page. [Sev2][WCAG1.3.1] 
40: Accessibility] [Screen reader -Playground]: Multiple ‘H1’ headings are defined in the page. [Sev3][WCAG2.4.6]
41: [Accessibility] [Supporting the platform -TSDocs]: TSDocs link is not visible properly in high-contrast dessert mode. [Sev3]
42: [Accessibility] [Supporting the platform -TSDocs]: Focus indicator is not moving to the displayed search results in high-contrast mode.
43: [Accessibility] [Supporting the platform -Tags]: Focus indicator is not visible on the “Improve this page” link in high-contrast mode. [Sev3]

**Comment by octogonz** [comment]
This issue has been closed because this Jekyll project has been retired. The documentation content has been migrated to Docusaurus. 

If the problem has not been addressed, please file a new issue under this GitHub project instead:  

https://github.com/microsoft/rushstack-websites
